app/controllers/profiles_controller.rb

```diff
@@ -44,6 +43,6 @@ def profile_params
       :user_id, :picture, :profession, :monday,
       :tuesday, :wednesday, :thursday, :friday, :avatar,
       contact_attributes
-      )
+    )
```

Comment:
I know, it's unrelated. But rubocop hassles us every time and again. And it should have never been committed un-linted anyway...